api/v2/routes_account_test.go

```diff
@@ -0,0 +1,220 @@
+package v2
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/RTradeLtd/Temporal/mocks"
+	"github.com/RTradeLtd/config"
+	"github.com/RTradeLtd/database/models"
+)
+
+func Test_API_Routes_Account(t *testing.T) {
```

Comment:
this test seems long and brittle, it might be a good idea to break it down and move to a more case-driven model

Reply:
Good idea.

Reply:
this test seems to still be really long and brittle? by case-driven i mean table-driven

it's pretty hard to read, and tests a lot of functionality in one go (**10** endpoints)